types/sync-storage/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for sync-storage 0.4
+// Project: https://github.com/raphaelpor/sync-storage
+// Definitions by: Dannomac <https://github.com/Dannomac>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/*
+*  Init prepares the SyncStorage to work synchronously, by getting all values for all keys previously saved on RN AsyncStorage.
+*/
+export function init(): any;
+/*
+*  Returns value of key
+*/
+export function get(key: string): any;
```

Comment:
Can this be more exact than `any`? I imagine local storage can really only return strings, unless this package does automatic type casting

Reply:
That works then